### ISSUE TITLE
Image resizing (and cropping?)

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#634]
I would like to add a new module about image resizing. There is much to learn, e.g. in terms of which interpolation one should use.

I wonder: should one also include image **cropping** into this "resizing: module? I would say it is conceptually different but at the same time, technically, one obtains an image of a "different size", so it could fit. The main reason I am pondering about including cropping into resizing is that I don't think there is much to teach about cropping; I find it so easy that I would not think it deserves an own module, or am I missing something?

I would love to read your opinions!

ping @stemarcotti @AnniekStok @maulakhan @tibuch @manerotoni @grinic

**Comment by tischi** [comment]
In fact, unsure now, maybe "image resizing" should be part of an ["image similarity transformation" module](https://github.com/NEUBIAS/training-resources/issues/635)?!!

**Comment by manerotoni** [comment]
I can add only from the motivation side:
to prepare data for Deep learning (or other algorithms) you do a lot of cropping (=tiling) and stitching.
To make the patches of images of an identical dimension you use also quite a bit of padding and resizing

If you use cropping as tiling of a large image, this is not that trivial and you would like to do it in a clever way. 

Don't forget resizing of labeled images that also need some different approach.

**Comment by tischi** [comment]
I see, good points, image cropping, tiling, chunking could be one module then...?

**Comment by manerotoni** [comment]
Yes, I think it something needed in particular if you have very large data sets. 
Try to pus 1 TB on your GPU. 
In a module you will obviously not work with such a large data set
